var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -58,8 +57,8 @@ class Namd(MakefilePackage):
 
     def _copy_arch_file(self, lib):
         config_filename = 'arch/{0}.{1}'.format(self.arch, lib)
-        shutil.copy('arch/Linux-x86_64.{0}'.format(lib),
-                    config_filename)
+        install('arch/Linux-x86_64.{0}'.format(lib),
```

Comment:
non-linux systems?

Reply:
Not supported 😃 

Anyway, that's something that can be saved for another PR.

Reply:
Agreed, but we should probably add `conflicts`-statements at some point